bigbluebutton-html5/imports/startup/server/logger.js

```diff
@@ -1,15 +1,30 @@
 import { Meteor } from 'meteor/meteor';
 import Winston from 'winston';
 
-let Logger = new Winston.Logger();
-
-// Write logs to console
-Logger.add(Winston.transports.Console, {
-  prettyPrint: false,
-  humanReadableUnhandledException: true,
-  colorize: true,
-  handleExceptions: true,
-});
+let Logger = new (Winston.Logger)({
```

Comment:
What are the reasons behind changing the construct syntax of the logger? The `new` syntax looks way cleaner.